myia/compile/debug_lin.py

```diff
@@ -52,6 +52,9 @@ def ref(n):
     outputs = get_outputs(lst, lst[0].graph.manager.uses, set(eqv.keys()))
     g.output = make_tuple((eqv[o] for o in outputs), g)
 
+    # Clone in case g contains subgraphs that have a different manager
+    g = clone(g)
```

Comment:
It was when I tried to make an example with `switch`, but then there was another problem with it returning a Graph instead of a numerical reference the VM could jump to.